deploy/main.js

```diff
@@ -12,9 +14,39 @@ const port = process.env['EXPRESS_PORT'] || 9000;
 
 const migMetaStr = fs.readFileSync(migMetaFile, 'utf8');
 const migMeta = JSON.parse(migMetaStr);
+
+const sanitizeMigMeta = (migMeta) => {
```

Comment:
I am assuming this is just to strip the secret from Pod logs?

Reply:
@pranavgaikwad yeah pretty much. This just strips the secret out of whats passed in to be consumed and visible in the JS.